.github/workflows/deploy-gcp-tests.yml

```
@@ -373,19 +381,23 @@ jobs:
       # (The cached state and disk are usually the same size,
       # but the cached state can be smaller if we just increased the disk size.)
       - name: Create ${{ inputs.test_id }} Docker volume
-        id: compute-ssh
-        uses: google-github-actions/ssh-compute@v0.1.2
-        with:
-          instance_name: ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}
-          zone: ${{ env.ZONE }}
-          ssh_private_key: ${{ secrets.GCP_SSH_PRIVATE_KEY }}
-          command: |
-            sudo e2fsck -v -f -p /dev/sdb \
-            && \
-            sudo resize2fs -p /dev/sdb \
-            && \
-            sudo docker volume create --driver local --opt type=ext4 --opt device=/dev/sdb \
-            ${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}
+        run: |
+          gcloud compute ssh \
+          github-service-account@${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --ssh-flag="-o ConnectionAttempts=20" \
+          --ssh-flag="-o ConnectTimeout=5" \
+          --command \
+          "\
+          sudo e2fsck -v -f -p /dev/sdb \
+          && \
+          sudo resize2fs -p /dev/sdb \
+          && \
```

If resizing still fails, here is a fix:

```suggestion
          || \
          echo 'resizing disk failed: ignoring this error, because resizing is only needed when CI fails because the disk is full'
```

---

I'll follow-up with this in https://github.com/ZcashFoundation/zebra/pull/5371